subsys/bluetooth/mesh/lpn.c

```diff
@@ -326,7 +326,7 @@ static int send_friend_req(struct bt_mesh_lpn *lpn)
 	struct bt_mesh_net_tx tx = {
 		.ctx = &ctx,
 		.src = bt_mesh_primary_addr(),
-		.xmit = POLL_XMIT,
+		.xmit = bt_mesh_net_transmit_get(),
```

Comment:
Could you elaborate why Friend Request should be sent with xmit value from network transmit count state?

Reply:
Yep. This is kind of `active scanning` that LPN performs on the primary subnetwork with `all friends` broadcast address. This doesn't follow the friendship timing parameters. I guess, this should be sent with reliability that entire network has been configured since this is primary network communication.

Reply:
Though specification doesn't provide any restriction for it.

Reply:
This is not good from the power consumption perspective as LPN will have to consider the adv duration in the receive delay. As you may see, when `CONFIG_BT_MESH_LPN_ESTABLISHMENT` is enabled, scanner is disabled during the friend request receive window. When the network transmit count state is small, this won't make big difference considering the friend receive window. Though with fine tune we could save some power here as well, but up to you. Note also that in nRF5 SDK for Mesh we also sent only 1 Friend Request without retransmissions.

Reply:
Actually, there is no universal good solution here. If a network doesn't have intensive communication then a single Friend Request is the power save. If a network has intensive communication then the network xmit setting will help save power due to LPN will try establishing again only after 1.1 seconds.
That will consume additional power as well.